### DISCUSSION TITLE
Possibility to create Workspace/Request collection Templates

### DISCUSSION THREAD

**Discussion by matthijssn** [Kong/insomnia#3838] [Ideas]
In my team we are using Insomnia to create API projects per Customer. It would be great when we can create a template structure with standard Environment jsons (By example: Test, Acceptance, Production) . In this way when you select the template as a base for a new collection, the basic structure of the collection is already there.

**Comment by dimitropoulos** [comment]
What is it about the (existing) ability to duplicate request collections that doesn't work for you?  Could you just create your template request collection and then duplicate it every time you want to create a new instance?

**Reply by matthijssn** [reply]
That can be a possibility... Have to look into that.

**Reply by dimitropoulos** [reply]
Ok, cool.  Please let me know how it works for you.
